Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -308,6 +308,12 @@ RDKIT_FILEPARSERS_EXPORT RWMol *PDBFileToMol(const std::string &fname,
                                              unsigned int flavor = 0,
                                              bool proximityBonding = true);
 
+RDKIT_FILEPARSERS_EXPORT RWMol *MMCifFileToMol(const std::string &fname,
```

Comment:
Based on the docs it looks like the format is `mmCIF`, so this probably should be:
```suggestion
RDKIT_FILEPARSERS_EXPORT RWMol *mmCIFFileToMol(const std::string &fname,
```